dataplug-facebook/conf/application.test.conf

```diff
@@ -5,7 +5,7 @@ silhouette {
   # Authenticator settings
   authenticator.cookieName = "authenticator"
   authenticator.cookiePath = "/"
-  authenticator.secureCookie = false // Disabled for testing on localhost without SSL, otherwise cookie couldn't be set
+  authenticator.secureCookie = false // Disabled for testing on localhost without SSL, otherwise cookie could not be set
```

Comment:
This change is to help my editor highlight correctly.  It appears three times per .conf file.